ecmd-core/capi/ecmdDataBufferBase.C

```diff
@@ -866,6 +886,10 @@ uint32_t ecmdDataBufferBase::shiftRight(uint32_t i_shiftNum, uint32_t i_offset)
     ETRAC2("**** ERROR : ecmdDataBufferBase::shiftRight: i_offset %d > NumBits (%d)", i_offset, iv_NumBits);
     RETURN_ERROR(ECMD_DBUF_BUFFER_OVERFLOW);
   }
+  if (i_shiftNum+i_offset > iv_NumBits) {
```

Comment:
I could see where this behavior is desired if you want to shift part of the data out of the buffer

Reply:
In this case a clearbit message is posted to the screen for the user, and the databuffer has this error logged against it.  By checking here, the user is told its from shiftRight instead of clearbit.  To error out, or not to error out?